README.md

```diff
@@ -43,10 +54,14 @@ Some additional options that most users do not need are hidden behind the `--adv
 the `user_configuration.json` contains all general installation configuration, whereas the `user_credentials.json`
 contains the sensitive user configuration such as user password, root password, and encryption password.
 
-An example of the user configuration file can be found here
-[configuration file](https://github.com/archlinux/archinstall/blob/master/examples/config-sample.json)
-and an example of the credentials configuration here
-[credentials file](https://github.com/archlinux/archinstall/blob/master/examples/creds-sample.json).
+### ⚙️ An Example File
```

Comment:
Change to `Example configuration files`

Reply:
I'd also like to avoid emoji's, if possible.